ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -124,6 +124,7 @@
 PTF_BP_IF_TEMPLATE = 'ptf-%s-b'
 ROOT_BACK_BR_TEMPLATE = 'br-b-%s'
 PTF_FP_IFACE_TEMPLATE = 'eth%d'
+OVS_INTERCONNECTION_BRIDGE_TEMPLATE = 'br-%s-ic-%s'
```

Comment:
Because a bridge name can't be longer than 15 chars, can you update this template to something like `bic-%s-%s` to lower the risk of reaching the bridge name limitation?

Reply:
Done